Example Apps/Example Swift/MenuManager.swift

```diff
@@ -206,6 +207,55 @@ private extension MenuManager {
             })
         })
     }
+
+    /// Menu item that shows remote control example
+    ///
+    /// - Parameters:
+    ///      - manager: The SDL Manager
+    ///      - remoteManager: The manager for controling and viewing remote control data
+    /// - Returns: A SDLMenuCell object
+    class func menuCellRemoteControl(with manager: SDLManager, remoteManager: RemoteControlManager) -> SDLMenuCell {
+        let remoteControlIcon = SDLArtwork(image: UIImage(named: RemoteControlIconName)!.withRenderingMode(.alwaysTemplate), persistent: true, as: .PNG)
+
+        // Clicking on cell shows alert message when remote control permissions are disabled
+        if (!remoteManager.isPermissionEnabled) {
```

Comment:
```suggestion
        guard remoteManager.isPermissionEnabled else {
```

This is a good candidate for a `guard`